apps/web/src/pages/templates/workflow/DigestMetadata.tsx

```diff
@@ -1,79 +1,25 @@
 import { Accordion, Group, useMantineColorScheme } from '@mantine/core';
 import { Controller, useFormContext } from 'react-hook-form';
-import { DigestTypeEnum, DigestUnitEnum } from '@novu/shared';
+import { DigestTypeEnum } from '@novu/shared';
 
 import { When } from '../../../components/utils/When';
-import { colors, Input, SegmentedControl, Select, Tooltip } from '../../../design-system';
-import { inputStyles } from '../../../design-system/config/inputs.styles';
-import { WeekDaySelect } from './digest/WeekDaySelect';
+import { colors, Input, Select, Tooltip } from '../../../design-system';
 import { Bell, Digest, Timer } from '../../../design-system/icons';
 import { TypeSegmented } from './digest/TypeSegment';
-import { IntervalSelect } from './digest/IntervalSelect';
-import { BackOffFields } from './digest/BackOffFields';
 import { WillBeSentHeader } from './digest/WillBeSentHeader';
-import { ScheduleMonthlyFields } from './digest/ScheduleMonthlyFields';
-import { useEffect } from 'react';
 import { RegularInfo } from './digest/icons/RegularInfo';
-import { format } from 'date-fns';
+import { TimedDigestMetadata } from './TimedDigestMetadata';
+import { RegularDigestMetadata } from './RegularDigestMetadata';
 
-const convertUnitToLabel = (unit: DigestUnitEnum) => {
-  switch (unit) {
-    case DigestUnitEnum.SECONDS:
-      return 'second';
-    case DigestUnitEnum.MINUTES:
-      return 'minute';
-    case DigestUnitEnum.HOURS:
-      return 'hour';
-    case DigestUnitEnum.DAYS:
-      return 'day';
-    case DigestUnitEnum.WEEKS:
-      return 'week';
-    case DigestUnitEnum.MONTHS:
-      return 'month';
-  }
-};
-
-export const DigestMetadata = ({ control, index, readonly }) => {
-  const {
-    formState: { errors, isSubmitted },
-    watch,
-    trigger,
-    setValue,
-  } = useFormContext();
+export const DigestMetadata = ({ index, readonly }: { index: number; readonly: boolean }) => {
```

Comment:
this one was split to separate components: `TimedDigestMetadata` and `RegularDigestMetadata`